uaclient/snap.py

```diff
@@ -93,3 +93,33 @@ def get_config_option_value(key: str) -> Optional[str]:
         return out.strip()
     except exceptions.ProcessExecutionError:
         return None
+
+
+class SnapPackage:
+    def __init__(self, name, version, rev, tracking, publisher, notes):
+        self.name = name
+        self.version = version
+        self.rev = rev
+        self.tracking = tracking
+        self.publisher = publisher
+        self.notes = notes
+
+
+def get_installed_packages() -> List[SnapPackage]:
+    out, _ = system.subp(["snap", "list"])
+    apps = out.splitlines()
+    apps = apps[1:]
+    snaps = list()  # type: List[SnapPackage]
```

Comment:
I'd use 
```suggestion
    snaps = []
```
but feel free to ignore it.

Reply:
I was trying to avoid some mypy errors, which, apparently, never existed.

Reply:
I see
So
when you create an empty list and just append things to it, the first append tells mypy which kind of list it is. Appending something different later would give you an error, but here you're fine